lms/djangoapps/teams/static/teams/js/spec/views/team_profile_header_actions_spec.js

```diff
@@ -0,0 +1,254 @@
+define([
```

Comment:
For Reviewers: I've renamed this file and added `EditButton` spec. Also, i've moved some common helper methods to the main spec `TeamProfileHeaderActionsView ` so we can use them in `JoinButton` & `EditButton`